### ISSUE TITLE
I often repeat myself, I often repeat repeat

### ISSUE THREAD

**Issue by BlackCapCoder** [PoshYeti/Zork-Clone---Zachorian-Adventure#1]
This would be a lot nicer if you separated the game content away from the game logic. I suggest you make one or more files for the actual game, and read them upon program execution.

XML might be a good and simple choice for this, for instance:

```xml
// cabin.xml

<?xml version="1.0"?>
<map>
  <name>Cabin</name>
  <location>
    <x>1</x>
    <y>-1</y>
  </location>
  <choices>
    <choise>
      <input>READ NOTE</input>
      <output>
        John, if you find this letter, take these coins
        Go to the shop and get yourself some dinner
        You find 4 coins underneath the doormat
        Coordinates for  the shop have been updated in your notebook
      </output>
      <addCoins>4</addCoins>
    </choise>

    <choise>
      <input>ENTER</input>
      ...
    </choise>
  </choices>
</map>
```

**Comment by PoshYeti** [comment]
You're quite right. Separating the game content from the game logic would make for an easier read. This project was mainly used to learn about the different switches, while loops, if statements and method calls in java/blueJ. The future versions of this game, will have more classes and separate the code in logical structures which allows for an easier read, and more optimised code.
But I must say, you're suggestion of an XML sounds like something i should really look into

**Comment by BlackCapCoder** [comment]
Speaking of, I noticed that you made a c# version. In c# (de)serialization to xml is very painless through the [serialization attribute](https://msdn.microsoft.com/en-us/library/ms973893.aspx). Thus, serialization to XML is pretty common in windows games. For instance, this is how [Amnesia](http://store.steampowered.com/app/57300/Amnesia_The_Dark_Descent/) saves your game.
